Common/Data/UniverseSelection/OptionChainUniverse.cs

```diff
@@ -99,15 +99,19 @@ public override IEnumerable<Symbol> SelectSymbols(DateTime utcTime, BaseDataColl
                 return Unchanged;
             }
 
-            var availableContracts = data.Data.Select(x => x.Symbol);
+            // Temporary: this can be removed when future options universe selection is also file-based
+            var availableContractsData = data.Symbol.SecurityType != SecurityType.FutureOption
+                ? data.Data.Cast<OptionUniverse>()
+                : data.Data.Select(x => new OptionUniverse() { Symbol = x.Symbol, Underlying = data.Underlying, Time = x.Time });
```

Comment:
Should set time & endtime I think?👌

Reply:
OptionUniverse` set EndTime on its own 
![image](https://github.com/user-attachments/assets/9d7c5d19-24eb-45dc-9188-3381869ed2bc)